transports/rabbitmq/delayed-delivery.md

```diff
@@ -21,58 +21,111 @@ N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.N.destination
+queue1(Queue: nsb.delay-level-01)
+exchange0(Exchange: nsb.delay-level-00)
+queue0(Queue: nsb.delay-level-00)
+delivery(Exchange: nsb.delay-delivery)
+
+exchange27 -->|"Binding: 1.#<br/>(No * wildcards)"| queue27
+exchange27 -->|"Binding: 0.#<br/>(No * wildcards)"| exchange26
+queue27 -->|"TTL: 2^27s ≈ 4.25 years"| exchange26
+
+exchange26 -->|"Binding: *.1.#<br/>(1 * wildcard)"| queue26
+exchange26 -->|"Binding: *.0.#<br/>(1 * wildcard)"| exchange25
+queue26 -->|"TTL: 2^26s ≈ 2.13 years"| exchange25
+
+exchange25 -.-> hiddenExchanges
+hiddenExchanges -.-> exchange1
+
+exchange1 -->|"Binding:<br/>*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.1.#<br/>(26 * wildcards)"| queue1
+exchange1 -->|"Binding:<br/>*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.0.#<br/>(26 * wildcards)"| exchange0
+queue1 -->|TTL: 2^1s = 2 seconds| exchange0
+
+exchange0 -->|"Binding:<br/>*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.1.#<br/>(27 * wildcards)"| queue0
+exchange0 -->|"Binding:<br/>*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.0.#<br/>(27 * wildcards)"| delivery
+queue0 -->|TTL: 2^0s = 1 second| delivery
+
+classDef exchangeClass stroke:#000000,stroke-width:2px,font-weight:bold;
+class exchange27,exchange26,exchange25,exchange1,exchange0,delivery exchangeClass
+
+classDef hiddenExchanges font-style:italic,fill:transparent,stroke:transparent;
+class hiddenExchanges hiddenExchanges
 
 end
 ```
 
-The delay levels are connected in this manner, from highest (27) to lowest (0). Each delay level's routing key's add wildcards as needed so that they are looking at the portion of the message's routing key that corresponds to its delay level.
+To avoid unnecessary traversal through the delay infrastructure, the message is published to the first applicable exchange by identifying the first level that will route to a queue, or in other words, the first `1` in the routing key. In the 10-second example (binary `1010`), the message would first be published to the `nsb.delay-level-03` exchange, and all of that exchange's bindings are evaluated. Only two bindings exist for the exchange, so either:
+
+* If the value is a `1`, the exchange will route the message to the `nsb.delay-level-03` queue, where it will wait until the TTL expires (2^3 seconds) before being forwarded to the `nsb-delay-level-02` exchange.
+* If the value is a `0`, the exchange will route the message to the `nsb.delay-level-02` exchange.
 
 
 ### Delivery
 
-The final delay-level exchange is bound to the delivery exchange instead of another delay level. Every endpoint that can receive a delayed message will have a binding to this exchange with a routing key corresponding to the endpoint's name, so the message will be delivered to the endpoint's queue.
+At the end of this process, the message is delivered to the `nsb.delay-delivery` exchange. It is at this final step where the message destination is evaluated to determine where the message should be routed to.
```

Comment:
Still think "routed" would be better than "delivered" here since we're still talking about actual RabbitMQ concepts.